src/Stores/Feedback/reducer.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+import type { Action } from '../../types';
+
+export type State = {
+  +message: ?string,
+};
+
+const initialState = {
+  message: null,
+};
+
+const feedbackReducer = (state: State = initialState, action: Action): State => {
+  switch (action.type) {
+    case 'COMMON_CLEAR_MESSAGE':
```

Comment:
Maybe change the names of the types as well from common to feedback?

Reply:
good catch!